autogen.sh

```diff
@@ -1,8 +1,19 @@
-#!/bin/sh
+#!/bin/bash
 
 rm -rf autom4te.cache
 mkdir -p config/m4 config/aux external/jemalloc
 
+GIT_FOUND=1
+command git --help 2>&1 > /dev/null || { \
```

Comment:
This is pretty much "religious" argument. I actually prefer the "textbook" syntax `2>&1`